lib/src/api_client.dart

```diff
@@ -108,6 +108,14 @@ class _ApiRequest {
           Uri.parse(url),
           headers: headers,
         );
+        break;
+      case _RequestMethod.post:
+        result = _client.post(
+          Uri.parse(url),
+          headers: {'Content-Type': 'application/json'},
```

Comment:
지금 작성해준 코드는 headers값을 기입했어도 post일 땐 모든게 무시되고 `{'Content-Type': 'application/json'}`로 고정으로 박히는 무서운 코드라 이부분은 매개변수를 바라보는 코드로 해주시구요.
```suggestion
          headers: headers,
```

Reply:
옹 suggestion 엄청 편하네요!!!!